resolvers/index.js

```diff
@@ -82,6 +85,16 @@ const mutationResolvers = {
       data: { motive }
     })
 
+    // grant user discourse badge
+    try {
+      await discourse.badges.assignBadgeToUser({
+        badge_id: process.env.DISCOURSE_BADGE_ID,
+        username: user.username
+      })
+    } catch (e) {
+      Sentry.captureException(e)
```

Comment:
Would be this the best place to track this type of things?
Why not capture the exception within the `discourse-node-api`?

Reply:
I think we should leave this to the user, because we don't know what they will use within the catch block to report/log the error, they can ignore it altogether for example.